app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,61 @@
+a {
+  &.disabled {
+    color: $gray-ccc;
+    cursor: not-allowed;
+    text-decoration: none;
+  }
+}
+
+body {
+  font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
+}
+
+a, a:hover, a:focus {
+  text-decoration: none;
+  border-bottom: 1px dotted $sul-link-color-border;
+}
+
+h1 {
+  font-family: 'dejavu_sansextralight';
+  color: $sul-h1-font-color;
+  padding: 10px 0;
+  letter-spacing: -0.05em;
+}
+
+h2 {
+  letter-spacing: 0;
```

Comment:
Properties should be ordered border-bottom, color, letter-spacing, line-height, margin